css/properties/masonry-auto-flow.json

```diff
@@ -1,40 +0,0 @@
-{
-  "css": {
-    "properties": {
-      "masonry-auto-flow": {
```

Comment:
This is at least partially supported in Firefox behind the `layout.css.grid-template-masonry-value.enable` flag.